changelog/unreleased/features/6886.md

```diff
@@ -0,0 +1,3 @@
+- Added custom header x-mapbox-app-info to requests.
+- Fix upcoming route alerts incorrect position.
+- Stabilized routes with alternatives refresh / re-route on deviate.
```

Comment:
What does it mean "stabilized", could we get more specific on what issue was resolved?

Reply:
Removed this entry.